docs/modules/ROOT/pages/reference/extensions/kamelet.adoc

```diff
@@ -37,3 +37,46 @@ Or add the coordinates to your existing project:
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Usage
+
+=== Pre-load Kamelets at build-time
+
+This extension allow to pre-load a set of Kamelets at build time through the following two options. The kamelet to pre-load can be listed using the `quarkus.camel.kamelet.names` property.
```

Comment:
Are there two options? Or just the one config property?

Reply:
Changed to reflect real option name.